packages/commonwealth/client/scripts/views/pages/CommunityManagement/Contests/useCommunityContests.ts

```diff
@@ -0,0 +1,33 @@
+import { useQuery } from '@tanstack/react-query';
+
+import { useCommunityStake } from 'views/components/CommunityStake';
+
+import mockedContests from './mockedContests';
+
+const useCommunityContests = () => {
+  const { stakeEnabled } = useCommunityStake();
+
+  const { data: contestsData, isLoading: isContestDataLoading } = useQuery<
+    typeof mockedContests
+  >({
+    queryKey: ['contests'],
```

Comment:
lets keep the query key in api config

Reply:
as per PR description `created temporary hook that pretends API call to fetch list of contests`. Sorry for not making it clear enough. Basically this is not a final solution as we are gonna probably use trpc approach here. Waiting for the backend changes.